### ISSUE TITLE
Can't open or create .Net Interactive notebook

### ISSUE THREAD

**Issue by dcuccia** [dotnet/interactive#1872]
On a clean Windows 11 install of VS Code (or Insiders), and installation of .Net Interactive notebooks, I can't create or open .Net Interactive notebooks. I'm not sure if this is different from #1813.

.Net Interactive 1.0.2462010 is the latest version that opens a UI when I select "Create a new .Net Interactive Notebook". Newer versions just hang with the message "Activating Extensions..." which seems to time out after a couple seconds:

![image](https://user-images.githubusercontent.com/479617/151475738-13803cd3-282e-4535-bdd6-3aa0b3d5a39f.png)

But even if a notebook (.dib or .ipynb) is created with the earlier versions, I can't seem to execute any C# code - kernel execution hangs on a Hello World.

Tried installing the .Net 5 SDK, just in case, but that didn't help. Here's my full set of .Nets:

```
PS C:\> dotnet --list-sdks
3.1.416 [C:\Program Files\dotnet\sdk]
5.0.404 [C:\Program Files\dotnet\sdk]
6.0.101 [C:\Program Files\dotnet\sdk]
6.0.200-preview.22055.15 [C:\Program Files\dotnet\sdk]
```

Repro'd on VS Code:

Version: 1.63.2 (user setup)
Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3
Date: 2021-12-15T09:40:02.816Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.22000

...and VS Code Insiders:

Version: 1.64.0-insider (user setup)
Commit: c6e58d126f6b5676242e32b34180b2a595f29b51
Date: 2022-01-14T05:19:33.429Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.22000

What can I do to help track down? Should I create a separate ticket? I've grown so comfortable using .Net Interactive in my daily workflow, I'm at a loss for what to do!

- OS
    - [ x] Windows 11
    - [ ] Windows 10
    - [ ] macOS
    - [ ] Linux (Please specify distro)
    - [ ] iOS
    - [ ] Android
 - Browser 
    - [ ] Chrome
    - [ ] Edge
    - [ ] Firefox
    - [ ] Safari
 - Frontend
    - [ ] Jupyter Notebook
    - [ ] Jupyter Lab
    - [ ] nteract
    - [ x] Visual Studio Code 
    - [x ] Visual Studio Code Insiders
    - [ ] Visual Studio
    - [ ] Other (please specify)

**Comment by profix898** [comment]
Same here. Can't use any .NET Interactive at the moment ... I'm on VS Code 1.63.2. I noticed that both of us have a 6.0.200-preview version installed (in addition to latest 6.0.100).

**Comment by jonsequitur** [comment]
Could you post the output of `dotnet --info`?

**Comment by dcuccia** [comment]
Absolutely. See below (just updated Visual Studio 2022 to 17.1.0 Preview 4.0) and the same issue persists:
```
PS C:\> dotnet --info
.NET SDK (reflecting any global.json):
 Version:   6.0.200-preview.22055.15
 Commit:    a3c3a2bf3b

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.22000
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.200-preview.22055.15\

Host (useful for support):
  Version: 6.0.1
  Commit:  3a25a7f1cc

.NET SDKs installed:
  3.1.416 [C:\Program Files\dotnet\sdk]
  5.0.404 [C:\Program Files\dotnet\sdk]
  6.0.101 [C:\Program Files\dotnet\sdk]
  6.0.200-preview.22055.15 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.22 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.1 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.22 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.22 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.12 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by profix898** [comment]
Mine looks almost identical, except that .NET 5 is not installed (otherwise same Windows version, same .NET SDKs, etc.)

**Comment by jonsequitur** [comment]
@dcuccia That's identical to mine and I'm not seeing any issues. Let's take a look at VS Code setup.

Have you customized anything in `settings.json` related to .NET Interactive? (I haven't.)

<img width="559" alt="image" src="https://user-images.githubusercontent.com/547415/151622545-1e7ea40c-ee07-4d07-9c41-2297018beab1.png">

Another possibility: I wonder if another VS Code extension might be interfering. Do you have any other notebook-related extensions installed? Here's what I'm running.

```console
> code --list-extensions
andriniaina.fsharp-interactive-datascience
anotherglitchinthematrix.monochrome
dhedgecock.radical-vscode
eamodio.gitlens
earshinov.filter-lines
GitHub.copilot
GitHub.remotehub
GitHub.vscode-pull-request-github
Ionide.Ionide-fsharp
mariorodeghiero.vue-theme
mel-brown.deep-purple
ms-dotnettools.csharp
ms-dotnettools.dotnet-interactive-vscode
ms-python.python
ms-python.vscode-pylance
ms-toolsai.jupyter
ms-toolsai.jupyter-keymap
ms-toolsai.jupyter-renderers
ms-vscode-remote.remote-wsl
ms-vsliveshare.vsliveshare
RandomFractalsInc.vscode-data-preview
RandomFractalsInc.vscode-data-table
samrapdev.outrun
tchayen.markdown-links
yzhang.markdown-all-in-one
```

**Comment by jonsequitur** [comment]
@brettfo 👀

**Comment by profix898** [comment]
I have the following VS Code extensions installed:
```
> code --list-extensions
christian-kohler.path-intellisense
cschlosser.doxdocgen
DotJoshJohnson.xml
eamodio.gitlens
EditorConfig.EditorConfig
formulahendry.code-runner
jeff-hykin.better-cpp-syntax
krvajalm.linter-gfortran
ms-dotnettools.csharp
ms-dotnettools.dotnet-interactive-vscode
ms-dotnettools.vscode-dotnet-runtime
ms-python.python
ms-python.vscode-pylance
ms-toolsai.jupyter
ms-toolsai.jupyter-keymap
ms-vscode-remote.remote-containers
ms-vscode-remote.remote-ssh
ms-vscode-remote.remote-ssh-edit
ms-vscode-remote.remote-wsl
ms-vscode.cmake-tools
ms-vscode.cpptools
ms-vscode.cpptools-extension-pack
ms-vscode.cpptools-themes
ms-vscode.powershell
ms-vsliveshare.vsliveshare
mutantdino.resourcemonitor
PKief.material-icon-theme
tht13.html-preview-vscode
vscode-icons-team.vscode-icons
vsls-contrib.codetour
yzhang.markdown-all-in-one
```

I can actually load an existing notebook file, but I noticed that C# is missing from the list of available languages for the code cells. It only has Markdown and Python (and it always falls back to Python when I try to execute the cell).

**Comment by dcuccia** [comment]
@jonsequitur no custom settings - brand new install:
![image](https://user-images.githubusercontent.com/479617/151627179-a86019dd-1a8e-4729-8eb9-e2f6b2b66e2f.png)

Here are my extensions (am I missing any I might need?):
```
PS C:\Users\dcucc> code --list-extensions
ms-dotnettools.csharp
ms-dotnettools.dotnet-interactive-vscode
ms-toolsai.jupyter
```

**Comment by jonsequitur** [comment]
@profix898 Which kernel are you seeing selected here?
![image](https://user-images.githubusercontent.com/547415/151627681-0ff3f079-8de7-4557-b6ef-094fad3a28ce.png)

**Comment by profix898** [comment]
@jonsequitur There is only Python 3.9.7 in the list ... Looks like .NET Interactive is not "registered" correctly.

Previously (before moving to Win 11 and fresh install) I always had Jupyter and the dotnet interactive global tool installed as well. But that should not be necessary, right? Will try anyway and report back. But, just VS Code + Extension + SDK should do it, right?

**Comment by dcuccia** [comment]
Not sure if this is helpful, but opening an existing .ipynb file with Powershell code does not give me a .Net Interactive kernel choice: 
![image](https://user-images.githubusercontent.com/479617/151629118-a1b6f93a-a06d-4708-b7a3-cc48a5c0a8ce.png)

**Comment by jonsequitur** [comment]
@profix898  The .NET Interactive Notebooks extension depends on the Jupyter extension. It doesn't depend on Python though.

**Comment by brettfo** [comment]
The root cause is here: dotnet/sdk#23687

The only workaround is to uninstall, or otherwise disable, the 6.0.200-preview.22055.15 sdk.

**Comment by dcuccia** [comment]
@brettfo bingo! repro'd that ticket's behavior. Unfortunately, I can't figure out a way to uninstall the 6.0.200-preview.22055.15 SDK, either through Version 17.1.0 Preview 4.0 individual components:

![image](https://user-images.githubusercontent.com/479617/151899346-1321569d-28ee-4c0f-896a-9ad6d129e957.png)

...or via dotnet-core-uninstall:

```
PS C:\>dotnet-core-uninstall list

This tool cannot uninstall versions of the runtime or SDK that are 
    - SDKs installed using Visual Studio 2019 Update 3 or later.
    - SDKs and runtimes installed via zip/scripts.
    - Runtimes installed with SDKs (these should be removed by removing that SDK).
The versions that can be uninstalled with this tool are:

.NET Core SDKs:
  6.0.101  x64    [Used by Visual Studio. Specify individually or use --force to remove]
  5.0.404  x64
  3.1.416  x64

.NET Core Runtimes:

ASP.NET Core Runtimes:
  5.0.13  x86
  5.0.13  x64

.NET Core Runtime & Hosting Bundles:
```

Assume this means I have to choose between .Net Interactive and VS 2022 preview bits, for now.

**Comment by dcuccia** [comment]
@jonsequitur is this resolved?

**Comment by jonsequitur** [comment]
I'm not able to repro this with the latest VS 2022 Previews installed. Are you still seeing it?

**Comment by dcuccia** [comment]
@jonsequitur you're right - I can't either, great news! Probably means [this](https://github.com/dotnet/sdk/issues/23687) wasn't the root cause after all?

**Comment by jonsequitur** [comment]
Probably not related, since .NET Interactive isn't using `dotnet run`.
